src/utils.ts

```diff
@@ -162,3 +162,13 @@ export const getUsername = (api: Gitlab) => {
     api.Users.showCurrentUser().then(currentUser => currentUser.username)
   )
 }
+
+export const once = <T>(fn: () => T) => {
+  let result: T | undefined
```

Comment:
Use a temporary `run = false` -> `run = true` check instead, then we don't need to return an empty string for invalid environment.

Reply:
Sure, that would also be fine.

Reply:
> As such, the once function is needless IMO?

Current solution is over developing IMO.

Reply:
Updated!